### DISCUSSION TITLE
Home Assistant legacy triggers

### DISCUSSION THREAD

**Discussion by michal-bartak** [Koenkk/zigbee2mqtt#12608] [General]
When `Home Assistant legacy triggers` option is enabled, the state of my Symfonisk controllers is reported to `devicename_action` topic. 

When disabled, no such topic exists, thus no related entity is created in HA. How can I intercept actions coming from the device in HA?
I use NodeRed for most automation. Tried to monitor all events passed from HA to NodeRed but found nothing related.

With regards

**Comment by Koenkk** [comment]
You can use the MQTT device trigger: https://www.zigbee2mqtt.io/guide/usage/integrations/home_assistant.html#via-mqtt-device-trigger-recommended

**Reply by michal-bartak** [reply]
Thank you for the info.
